app/assets/javascripts/discourse/lib/search.js.es6

```diff
@@ -43,6 +44,17 @@ export function translateResults(results, opts) {
     })
     .compact();
 
+  results.groups = results.groups
+    .map(group => {
+      const groupName = Handlebars.Utils.escapeExpression(group.name);
+      return Ember.Object.create({
```

Comment:
I'm curious if we're observing these attributes. If not, it's often faster to return a POJO.